lib/player.py

```diff
@@ -271,6 +271,9 @@ def seek(self, offset, settings_changed=False, seeking=SEEK_IN_PROGRESS):
 
         self.seeking = self.SEEK_IN_PROGRESS
 
+        if self.player.playState == self.player.STATE_PAUSED:
+            self.player.pauseAfterPlaybackResumed = True
```

Comment:
I think we can just stick with `self.player.pauseAfterPlaybackStarted` (well one or the other in any case) because it doesn't matter which event it is, we'll clear the flag anyway.

Reply:
Not really, I've seen the following when switching between qualities direct-play to direct-play:
- OnPlaybackStarted fired *and* OnPlaybackResumed fired: had to pause at both occasions to-repause
- either one fired individually without the other one being fired

Reply:
And using the same flag for both events ends up clearing the flag in Started, then the player resuming and not re-pausing, because Resumed doesn't see the flag as True.